### ISSUE TITLE
23.07.11 (화) Issue

### ISSUE THREAD

**Issue by SaFD-00** [SaFD-00/2023_RCV_SUMMER_URP#8]
# <공부한 내용>
## 1. SSD
- 성능평가지표(https://www.waytoliah.com/1491)
- Base Network
    1. Backbone: VGG16
    2. Pre-trained Dataset: ILSVRC CLS-LOC
    3. FC6, FC7 -> convolutional layers
    4. Pool5: 2x2 -> 3x3
    5. a trous algorithm을 이용해서 ”holes”을 채움
    6. Dropout layer, FC8 layer 제거
    7. Learning Rate: 10^(-3)
    8. Momentum: 0.9
    9. Weight Decay: 0.0005
    10. Batch Siz: 32
- Hard negative mining(in R-CNN)

## 2. None SSD
- 기존: Sliding Window
- R-CNN 계열(2-stage detector): classification과 localization 문제를 순차적으로 해결
- YOLO, SSD 계열(1-stage detector): classification과 localization 문제를 동시에 해결
		: YOLO VS SSD
			SSD 모델의 핵심적인 아이디어는 다양한 scale의 feature map을 사용한다는 점
			-> 기존의 모델은 convolutional network를 거친 단일한 scale을 가진 feature map을 detection을 위해 사용 			-> YOLO v1 모델의 경우 7x7(x30) 크기의 feature map만을 사용
			-> 하지만 이처럼 단일한 scale의 feature map을 사용할 경우, 다양한 크기의 객체를 포착하는 것이 어려움 			-> 이러한 문제를 해결하기 위해 SD network 중간에 존재하는 conv layer의 feature map들을 추출하여 detection시 사용하는 방법을 제안
